core/src/main/resources/db/functions/vw_econewswithtags.sql

```diff
@@ -0,0 +1,16 @@
+CREATE MATERIALIZED VIEW public.vw_econewswithtags
```

Comment:
Please add IF NOT EXISTS or the view will be recreated on each application startup.

Reply:
ok)